Gopkg.toml

```diff
@@ -267,3 +267,7 @@
 [[constraint]]
   name = "github.com/vishvananda/netns"
   revision = "0a2b9b5464df8343199164a0321edf3313202f7e"
+
+[[constraint]]
```

Comment:
I think you need to add this to a license file since it's a new dependency.

Reply:
The license of this package says `..As this is fork of the official Go code the same license applies.` and basically pastes the text of `Go` license there. From Google, it seems the `Go` license can be described as `BSD-style + patent grant`. Should I put it here or just skip it since we already use `Go` and this package is covered under that?